src/__test__/components/data-management/UploadDetailsModal.test.jsx

```diff
@@ -0,0 +1,119 @@
+import React from 'react';
+import {
+  render, screen, waitFor,
+} from '@testing-library/react';
+import '@testing-library/jest-dom';
+import userEvent from '@testing-library/user-event';
+import UploadDetailsModal from 'components/data-management/UploadDetailsModal';
+import UploadStatus, { messageForStatus } from 'utils/upload/UploadStatus';
+
+const mockOnDelete = jest.fn();
+const mockOnCancel = jest.fn();
+const mockOnDownload = jest.fn();
+const mockOnRetry = jest.fn();
+
+const defaultProps = {
+  visible: true,
```

Comment:
is `visible` prop still in use?

Reply:
i kept it just in case it could be used but ill remove it i guess since the decision to render the modal happens in the parent component normally.